supports-and-hinges/relconnectsrigidlink.md

```diff
@@ -2,379 +2,38 @@
+| :---------------------------: | :--------------: | :------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------: | :--------------------------------------------------------------------: | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+|              Name             |      String      |                                                                                                       RL1                                                                                                      |                                   yes                                  | Human readable unique name of the object                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 |
+|             Nodes             |      String      |                                                                                                     N3; N4                                                                                                     |                                   yes                                  | <p>The name of the valid existing node (<a href="../structural-analysis-elements/structuralpointconnection.md#node">StructuralPoinConnection</a>), or (<a href="../structural-analysis-elements/structuralcurvemember.md#1d-member-beam-column">StructuralCurveMember</a>, <a href="../structural-analysis-elements/structuralsurfacemember.md#2d-member-plate-wall">StructuralSurfaceMember</a>) for internal nodes</p><p>Maximum of two nodes can be connected by RelConnectsRigitLink</p><p>First in node is seen as "master node", second in row is "slave node"</p> |
```

Comment:
Whats this part about the structuralcurvemember and structuralsurfacemember for internal nodes?